notebooks/simple_xgboost_example.ipynb

```diff
@@ -0,0 +1,714 @@
+    "import numpy\n",
+    "from numpy import loadtxt\n",
+    "from xgboost import XGBClassifier\n",
+    "from sklearn.model_selection import train_test_split\n",
+    "from sklearn.metrics import accuracy_score\n",
+    "\n",
+    "import os\n",
+    "import signal\n",
+    "import subprocess"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Generate dummy data to perform binary classification\n",
+    "seed = 7\n",
+    "features = 9 # number of sample features\n",
+    "samples = 10000 # number of samples\n",
+    "X = numpy.random.rand(samples, features).astype('float32')\n",
+    "Y = numpy.random.randint(2, size=samples)\n",
+    "\n",
+    "test_size = 0.33\n",
+    "X_train, X_test, y_train, y_test = train_test_split(X, Y, test_size=test_size, random_state=7)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "model = XGBClassifier()\n",
+    "model.fit(X_train, y_train)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "y_pred = model.predict(X_test)\n",
+    "predictions = [round(value) for value in y_pred]\n",
+    "accuracy = accuracy_score(y_test, predictions)\n",
+    "print(\"Test Accuracy: %.2f%%\" % (accuracy * 100.0))"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Export and load XGBoost model in Triton Inference Server\n",
+    "\n",
+    "For deploying the trained XGBoost model in Triton Inference Server, follow the steps below:\n",
+    "\n",
+    "**1. Create a model repository and save xgboost model checkpoint:**\n",
+    "\n",
+    "We'll need to create a model repository that looks as follows:\n",
+    "\n",
+    "```\n",
+    "model_repository/\n",
+    "`-- fil\n",
+    "    |-- 1\n",
+    "    |   `-- xgboost.model\n",
+    "    `-- config.pbtxt\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Create directory to save the model\n",
+    "![ ! -d \"/data/model_repository\" ] && mkdir -p /data/model_repository/fil/1\n",
+    "\n",
+    "# Save your xgboost model as xgboost.model\n",
+    "model.save_model('/data/model_repository/fil/1/xgboost.model')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**2. Create and save config.pbtxt**\n",
+    "\n",
+    "To deploy the model in Triton Inference Server, we need to create and save a protobuf config file called config.pbtxt under `model_repository/fil/` directory that contains information about the model and the deployment. Sample config file is available here: [link](https://github.com/triton-inference-server/fil_backend#configuration)\n",
+    "\n",
+    "Essentially, the following parameters need to be updated as per your configuration\n",
```

Comment:
Let's try to signpost this a bit better. Tell them that we are going to generate the config file for them with the following command and then tell them that we are presenting the config here so that they know what they'll need to tweak if deploying a model of their own. Also, avoid "essentially." We should either be direct about what we're leaving out of discussion or be sure to include all relevant details.